docroot/modules/humsci/hs_bugherd/src/ConfigOverrides.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Drupal\hs_bugherd;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\encrypt\EncryptService;
+use Drupal\encrypt\Entity\EncryptionProfile;
+
+/**
+ * Example configuration override.
+ */
+class ConfigOverrides implements ConfigFactoryOverrideInterface {
```

Comment:
These override classes are super handy and perfect for this use case. Good choice.